cmd/context/use.go

```diff
@@ -147,6 +149,19 @@ func (c *ContextCommand) Run(ctx context.Context, ctxOptions *ContextOptions) er
 	return nil
 }
 
+// RunStateless is the fn to use until the refactoring of the context command itself if you want to make use
+// of an injected context instead of using the global context variable.
+func (c *ContextCommand) RunStateless(ctx context.Context, ctxOptions *ContextOptions) (*buildCtx.OktetoContext, error) {
```

Comment:
Should we return an `OktetoContext` that has all the methods and we only need the interface in order to get use of it? Right now `RunStateless` is returning a `buildCtx` object. If we want to have more commands make use of this, we should be returning an object from `context` package

Reply:
good point. I think we should return a general context okteto and have each command transform it to its specific context that will expose only the necessary methods. WDYT?